### PR TITLE
Cleanup run grow workflow solvers

### PR DESCRIPTION
**Pull request by FJanssen-TNO**
Created a separate function to set the solversettings when the problem is being called, preventing to have to create classes for every problem and solver combination. 
This allows clean up of the code where new classes are created to inherit the solversettings and the problem. 
It will allow for easier adaptation and running when other problemclasses and solvers are added.

Before merging to main, this should be checked with the integration on which classes are currently passed on how it will have to be done in the future.

Open for discussion:
Potentially we could leave some of the old HIGHS classes in the code temporarily with the warning that it is depreciated and will be removed in the future.

### CODE REVIEW

File: src/mesido/workflows/grow_workflow.py [hunk 1]
```diff
@@ -110,6 +110,24 @@ def solver_options(self):
         return options
 
 
+class SolverCPLEX:
```
Comment (KobusVanRooyen) [inline src/mesido/workflows/grow_workflow.py:113]: Mention this addition in the CGANGELOG as well

File: examples/PoCTutorial/src/run_grow_tutorial.py [hunk 2]
```diff
@@ -1,10 +1,10 @@
 from pathlib import Path
 
 from mesido.esdl.esdl_parser import ESDLFileParser
-from mesido.workflows import EndScenarioSizingStagedHIGHS, run_end_scenario_sizing
+from mesido.workflows import EndScenarioSizingStaged, run_end_scenario_sizing
```
Comment (KobusVanRooyen) [inline examples/PoCTutorial/src/run_grow_tutorial.py:4]: Add CHANGELOG

**Comment by KobusVanRooyen** [comment]
@FJanssen-TNO  review completed

File: src/mesido/workflows/grow_workflow.py (hunk 1 above)
Comment (FJanssen-TNO) [reply src/mesido/workflows/grow_workflow.py:113]: updated

File: examples/PoCTutorial/src/run_grow_tutorial.py (hunk 2 above)
Comment (FJanssen-TNO) [reply examples/PoCTutorial/src/run_grow_tutorial.py:4]: updated

**Review by KobusVanRooyen — APPROVED** [review]

**Comment by KobusVanRooyen** [comment]
@FJanssen-TNO approved

**Pull request merged**